content/en/docs/setup/upgrade/canary/index.md

```diff
@@ -121,36 +114,72 @@ istiod-canary-6956db645c-vwhsk
 
 The output confirms that the pod is using `istiod-canary` revision of the control plane.
 
-## Stable revision labels (experimental)
+## Stable revision labels
```

Comment:
Should you also change the feature status page (add it to the list as well)?

Reply:
I added "experimental" back to the heading pending TOC graduation of this feature to Alpha, and added the feature to `data/features.yaml`.